wot-wg-2023-draft.html

```diff
@@ -298,35 +276,24 @@ <h2>Scope</h2>
             Explain the semantics on how to version Thing Descriptions.
           </dd>
           <dt>
-            <strong>Normative TD Parsing, Consuming and Validation</strong>
+            <strong>Normative Linting, Parsing, Consuming and Validation mechanism for TDs</strong>
             (<a href="#thing-description-deliverable">Thing Description</a>,
             <a href="#scripting-deliverable">Scripting API</a>,
             <a href="#profiles-deliverable">Profiles</a>):
           </dt>
           <dd>
-            Define normative algorithms for parsing, consuming and validating Thing Descriptions.
-          </dd>
-          <dt>
-            <strong>Linting mechanism for TDs</strong>
-            (<a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#profiles-deliverable">Profiles</a>):
-          </dt>
-          <dd>
-            Define a linting mechanism for Thing Descriptions that allows adding additional rules.
+            Define a linting mechanism for Thing Descriptions that allows adding additional rules and normative mechanism for
+            parsing, consuming and validation of TDs for Consumers.
           </dd>
           <dt><strong>Binding Mechanism</strong>
             (<a href="#binding:-deliverable">Binding Templates</a>,
-            <a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#security-deliverable">Security</a>,
-            <a href="#profiles-deliverable">Profiles</a>):
-          </dt>
-          <dd>Add normative binding mechanism</dd>
-          <dt><strong>Protocol Bindings</strong>
-            (<a href="#binding:-deliverable">Binding Templates</a>):</dt>
-          <dd>Define bindings for specific protocols of interest.</dd>
-          <dt><strong>Payload Bindings</strong>
+              <a href="#thing-description-deliverable">Thing Description</a>,
+              <a href="#security-deliverable">Security</a>,
+              <a href="#profiles-deliverable">Profiles</a>):</dt>
+            <dd>Add normative binding mechanism</dd>
+          <dt><strong>Protocol and Payload Bindings</strong>
             (<a href="#binding:-deliverable">Binding Templates</a>):</dt>
-          <dd>Define bindings for payload formats of interest.</dd>
+          <dd>Define bindings for specific protocols and payloads of interest while focusing to support ecosystems.</dd>
```

Comment:
The word "ecosystem" is a bit vague and open to interpretation as what the "focus" actually is.

For example, I'm a bit sceptical that a "platform binding template" specifically for a proprietary platform like Philips Hue (the example given in the latest WoT Binding Templates draft) would actually help interoperability, since Consumers would have to specifically implement a separate binding for each device vendor.